iterator_massive.py

```diff
@@ -0,0 +1,34 @@
+class MassiveIterator:
```

Comment:
Why do we need this one?

Reply:
I'm not sure, I suppose we should create the iterator class without "next" which have no stage and another class which can save the stage. Perhaps, we can use MassiveIterator as a metaclass in the future, if we will want to create an iterator with the same parameters which behaviour is different from range behaviour.